Makefile

```diff
@@ -301,3 +301,9 @@ test-acceptance-generate-report:
 ## Serves acceptance tests report at http://localhost:8088
 test-acceptance-serve-report:
 	$(Q)CONTAINER_RUNTIME=$(CONTAINER_RUNTIME) $(HACK_DIR)/allure-report.sh serve
+
+.PHONY: release-manifests
+## prepare a manifest file for Release which installs the sbo operator on vanilla k8s
```

Comment:
how about: `prepare a manifest file for releasing operator on vanilla k8s cluster`?